doc/manual/src/protocols/tarball-fetcher.md

```diff
@@ -0,0 +1,40 @@
+# Serving Tarball Flakes
+
+Tarball flakes are served as regular tarballs via HTTP or the file
+system (for `file://` URLs).
+
+An HTTP server can return an "immutable" flakeref appropriate for lock
```

Comment:
As we're making use of existing facilities of the HTTP standard, I would not encourage people to use non-standard strings where a normal URL is expected. Furthermore accepting arbitrary flakerefs means that we'd have to care about redirects to local resources, which is a security risk.

```suggestion
An HTTP server can return an "immutable" HTTP URL appropriate for lock
```

Reply:
This affects the text in other places as well.

The term `flakeref` isn't accurate for something that may not be a flake.